packages/dev/src/utils/data.ts

```diff
@@ -1,5 +1,8 @@
 import { GenericDataRecord } from '@unovis/ts'
 import { sample } from './array'
+/* eslint-disable-next-line @typescript-eslint/no-var-requires, @typescript-eslint/naming-convention */
+const SeedRandom = require('seedrandom')
+export const rng = new SeedRandom('hello.')
```

Comment:
Maybe: `hello.` → `unovis` 🙂

Reply:
Also what `rng` stands for? I would try to pick a more explanatory name to make the code easier to read

Reply:
> Also what `rng` stands for? I would try to pick a more explanatory name to make the code easier to read

Random number generator. I thought it's pretty standard. I could update it to it's full name.